values.yaml

```diff
@@ -428,11 +428,47 @@ workflows:
       cpu: 1000m
       memory: 2048Mi
 
+# Config for optional standalone dbconnector deployment. 
 dbconnector:
+  # By default (disabled), the main backend makes resource queries directly; if
+  # enabled, only dbconnector pods will make resource queries on behalf of main
+  # backend and workflow-backend pods.
+  enabled: false
+
   java:
-    # Disable this to disable Retool's Java dbconnector.
+    # Disable this to disable Retool's Java dbconnector. Applies whether
+    # dbconnector.enabled is true or false.
     enabled: true
 
+  # Desired pod count for dbconnector deployment.
+  replicas: 1
+
+  # (Optional) If set, make dbconnector pods wait the specified time before
+  # kubernetes will forcibly kill them when they need to be rescheduled. Setting
+  # this to a long duration will minimize disruption to any long-running
+  # resource queries during updates, at the cost of making updates take longer.
+  # terminationGracePeriodSeconds: 960
+
+  # If necessary, specify the resources to provision the workflows-backend pod
+  # separately from the main backend pods. If unspecified, uses the same
+  # resources configured for main backend.
+  # resources:
+  #   limits:
+  #     cpu: 4096m
+  #     memory: 8192Mi
+  #   requests:
+  #     cpu: 2048m
+  #     memory: 4096Mi
+
+  config:
```

Comment:
did you mean to set this config intentionally? I'm surprised to see such a high default postgres max pool size, especially on the dbc deployment, as that would be specific to the Postgres connector which isn't global.